mod/notifications/views/default/notifications/subscriptions/collections.php

```diff
@@ -7,19 +7,17 @@
 $user = $vars['user'];
 
 //@todo JS 1.8: no ?>
-<script type="text/javascript">
-	
+<script>
 	function setCollection(members, method, id) {
```

Comment:
Goal here is minimum to get JS functional again. I just happened to clean up the script tags while I was at it.